Makefile

```diff
@@ -0,0 +1,19 @@
+SHELL=bash
+
+test:
+	go test -count=1 -mod=mod -race -cover ./...
+
+.PHONY: test
+
+audit:
+	go list -json -m all | nancy sleuth
+.PHONY: audit
+
+build:
+	go build -mod=mod ./...
+.PHONY: build
+
+.PHONY: lint
+lint:
+	exit
+
```

Comment:
```suggestion
SHELL=bash

.PHONY: test
test:
	go test -count=1 -mod=mod -race -cover ./...

.PHONY: audit
audit:
	go list -json -m all | nancy sleuth

.PHONY: build
build:
	go build -mod=mod ./...

.PHONY: lint
lint:
	exit

```

Reply:
I "think" I've amended this? (Can't really work out the differences!)